utils/vlog/tests/carry/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_subdirectory(cblock)
+add_file_target(FILE carry.sim.v SCANNER_TYPE verilog)
```

Comment:
Can you add a small comment (or a README) describing what features you are testing here?  Maybe in the verilog?  It will aid me in determining what your intention was, and determine if the test actually provides the coverage we wanted.

Reply:
@kgugala - Each of the tests should probably have a README.md file?

This test is checking v2x support for the carry pack pattern production thanks to the `(* carry *)` attribute on the `cblock` input / output ports.

v2x should correct;
 * Detect the `(* carry *)` attribute
 * Understand the port direction
 * Understand the need to add a pack pattern is needed in `carry.sim.v` as the start + end of a signal path both have `(* carry *)` attributes. Then generate a unique / useful name for the pack pattern.
 * Check that `(* carry *)` pins are not mixed onto nets which don't have the `(* carry *)` attributes.

Reply:
@kgugala

Reply:
Added readme for all the tests